AppCenter/AppCenter/MSAppCenter.m

```diff
@@ -577,12 +577,12 @@ - (void)initializeChannelGroup {
   @synchronized(self) {
 
     // Construct channel group.
-    if (self.appSecret) {
+    if (!self.appSecret && self.logUrl != kMSAppCenterBaseUrl) {
```

Comment:
Why do you check `kMSAppCenterBaseUrl`? What if the log URL is still for App Center but not PROD?

Reply:
App Center will always have the appSecret, so it will not execute this statement when the log url is for app center. 
When it initialize the app center instance, the log URL will always set as `kMSAppCenterBaseUrl`. 
If we only check the appSecret and don't check the `kMSAppCenterBaseUrl`, the default log URL will be changed to `kMSAppCenterBaseUrl` when app startups with one collector mode in original code.
I think it doesn't mase sence

Reply:
So, this is the scenario.
1. Call `setLogUrl` with valid App Center endpoint (not PROD one).
2. Start App Center.
In this case, we couldn't prevent the App Center custom endpoint being set to One Collector endpoint. So, it isn't necessary since it doesn't cover all scenarios. What do you think?

Reply:
Remove this extra check as discussed with @jaelim-ms